### DISCUSSION TITLE
How to build for Windows

### DISCUSSION THREAD

**Discussion by bowerhaus** [artisan-roaster-scope/artisan#1219] [Q&A] [answered]
Hi,

I can see that there is no Windows installer for the current continuous build (only the Windows legacy version).
Are there instructions anywhere for building the latest continuous build for Windows 11 from the source?

Thanks

Andy Bower

**Comment by poundy** [comment, marked as answer]
![image](https://github.com/artisan-roaster-scope/artisan/assets/279966/21e1ffa1-4577-41b9-9c04-9484a0603f4f)
Seems to be one?

**Reply by poundy** [reply]
possibly there was a build pipeline delay, since it was hours behind the other platforms... or someone unblocked whatever was causing that to be missed...

**Reply by bowerhaus** [reply]
Thanks for checking. I had just managed to work out how to drive Appveyor to create the builds but your reply beat me to it.

Thanks for the quick response.
